internal/binutils/testdata/build_binaries.sh

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash -x
```

Comment:
I think the shebang must be a first line to be recognized. In this case it will be ignored I think which means the script may be executed with non-bash shell and (I think) $OSTYPE is a bash-specific feature.

Reply:
good catch, thank you.